src/Block/Service/MenuBlockService.php

```diff
@@ -57,11 +57,11 @@ class MenuBlockService extends AbstractAdminBlockService
 
     /**
      * @param string                     $name
-     * @param EngineInterface            $templating
+     * @param Environment                $templating
      * @param MenuProviderInterface      $menuProvider
      * @param MenuRegistryInterface|null $menuRegistry
      */
-    public function __construct($name, EngineInterface $templating, MenuProviderInterface $menuProvider, $menuRegistry = null)
+    public function __construct($name, $templating, MenuProviderInterface $menuProvider, $menuRegistry = null)
```

Comment:
Can you replace parameters with typehint and remove the comment?

Reply:
@kunicmarko20 i am sorry, i did not understans what you mean =( you say, that i have to replace `$templating` with `Environment $templating`? But exception renderers contain this type hint and your comment is the same there.

Reply:
@covex-nn sorry, my bad, here you can skip it but on other places you can replace comment with typehint in constructor, right?